server/Startup.cs

```diff
@@ -17,18 +18,20 @@ public class Startup
         public void ConfigureServices(IServiceCollection services)
         {
             services.AddMvc();
+            services.AddScoped<ValidateModelStateAttribute>();
```

Comment:
I would I add clarification that ValidateModelStateAttribute is supposed to be used because of IActionResult presence in POST handler (if this actually correct). Particularly you cant use it for hello Get request handler method, can we?